icu4j/main/classes/core/src/com/ibm/icu/impl/units/UnitConverter.java

```diff
@@ -211,20 +211,24 @@ private void multiply(BigDecimal value, int power) {
             }
         }
 
-        public Factor applySiPrefix(MeasureUnit.SIPrefix siPrefix) {
+        /** Apply SI or binary prefix to the Factor. */
+        public Factor applyPrefix(MeasureUnit.MeasurePrefix unitPrefix) {
             Factor result = this.copy();
-            if (siPrefix == MeasureUnit.SIPrefix.ONE) {
+            if (unitPrefix == MeasureUnit.MeasurePrefix.ONE) {
                 return result;
             }
 
-            BigDecimal siApplied = BigDecimal.valueOf(Math.pow(10.0, Math.abs(siPrefix.getPower())));
+            Integer base = unitPrefix.getBase();
```

Comment:
Why capital-I `Integer` rather than `int`?

Reply:
Heh, "I have no idea": I can't remember what inspired this.

(I find one for-loop using "Integer" in this file - should I change that to "int" too? I don't think that one was my inspiration though!)